### DISCUSSION TITLE
Capture usage (token counts) for openai models via `stream_options: {"include_usage": true}`

### DISCUSSION THREAD

**Discussion by marcklingen** [open-webui/open-webui#5770] [Ideas]
Currently, usage `info` is only captured with stream=False for openai models. It would be useful to capture usage information also for streaming responses.

API reference: https://platform.openai.com/docs/api-reference/chat/create#chat-create-stream_options

**Comment by tjbck** [comment]
<img width="1228" alt="image" src="https://github.com/user-attachments/assets/92d105c8-be36-4949-a333-c65b1fec6fad">
<img width="521" alt="image" src="https://github.com/user-attachments/assets/120a4708-2055-4f11-8322-3641b2982e89">

**Comment by marcklingen** [comment]
thank you!
